src/logger/loggerTaskEx.c

```diff
@@ -246,8 +247,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* send the sample on to the telemetry task(s) */
-                if (sampledRate >= telemetrySampleRate ||
-                    currentTicks % telemetrySampleRate == 0)
+                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
```

Comment:
Example: configuring RCP for different sample rates (example, setting everything to 1Hz) and verifying bluetooth telemetry or cellular telemetry still works?